src/dbos-runtime/cli.ts

```diff
@@ -282,7 +313,15 @@ workflowCommands
   .option('-d, --appDir <string>', 'Specify the application root directory')
   .action(async (uuid: string, options: { appDir?: string; host: string }) => {
     const [dbosConfig, _] = parseConfigFile(options);
-    await restartWorkflow(options.host, uuid, getGlobalLogger(dbosConfig));
+    if (dbosConfig.databaseUrl === undefined) {
+      throw new Error('Database URL is not defined');
+    }
+    const client = await DBOSClient.create(dbosConfig.databaseUrl);
+    try {
+      await client.forkWorkflow(uuid, 0);
```

Comment:
Let's also print (in JSON) the status of the forked / resumed /restarted workflow

Reply:
forking/resuming/restarting will always lead to the WF being in the `ENQUEUED` state which might confuse folks if the original WF invocation wasn't queued. I'm open to changes here, but not in this PR.